script/bootstrap.sh

```diff
@@ -128,5 +128,5 @@ readonly CUR_DIR="$(dirname "$(realpath -s $0)")"
 readonly PROJECT_DIR=${CUR_DIR}/../
 
 cd ${PROJECT_DIR}
-git submodule update --init --recursive
+git submodule update --init --recursive --depth=1
```

Comment:
any reason for doing it ?

Reply:
This can speed up the submodule cloning process.

Reply:
Thanks. New skill got. :+1: